cypress/objects/task.js

```diff
@@ -0,0 +1,47 @@
+class task {
+    visitTasks() {
+        cy.visit('/tasks');
+    }
+    
+    getCreateTaskButton() {
+        return cy.get('[type="button"]');
+    }
+
+    getTaskName() {
```

Comment:
this is a bit confusing, but maybe try adding get**Random**TaskName this would bring more clarity to the functions purpose and actual behaviour